elements/lisk-cryptography/src/nacl/index.ts

```diff
@@ -14,20 +14,16 @@
  */
 import { NaclInterface } from './nacl_types';
 
-// tslint:disable-next-line no-let
-let lib: NaclInterface;
+// tslint:disable-next-line no-let no-require-imports no-var-requires
+let lib: NaclInterface = require('./slow');
 
 try {
-	if (process.env.NACL_FAST === 'disable') {
-		throw new Error('Use tweetnacl');
+	if (process.env.NACL_FAST !== 'disable') {
```

Comment:
the NACL_FAST must be a Boolean and fall back to false. There’s no environment variable in React Native.